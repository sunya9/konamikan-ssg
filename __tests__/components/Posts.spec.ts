import { mount } from '@vue/test-utils'
import { post } from '@/__tests__/fixtures/postOrPage'
import Posts from '@/components/Posts.vue'

describe('Posts', () => {
  it('normal', () => {
    expect(
      mount(Posts, {
        propsData: {
          posts: [post]
        }
      })
    ).toMatchSnapshot()
  })
})
