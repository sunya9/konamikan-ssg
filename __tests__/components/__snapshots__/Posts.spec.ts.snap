// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Posts normal 1`] = `
<ul>
  <li>
    <div class="oneline"><a><span class="has-text-grey">
          04/05
        </span> <span>title</span></a></div>
  </li>
</ul>
`;
